scripts/install_evmone.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env sh
```

Comment:
As long as this file is only meant to be run inside a certain docker container, maybe put the word `docker` in the script filename?

Reply:
This script will be run by the ossfuzz build CI to link against evmone libraries. This and other `install` scripts may be renamed like so
- `install_evmone.sh` -> `install_evmone_ci.sh`
- `install_libfuzzer.sh` -> `install_libfuzzer_ci.sh`
- `install_lpm.sh` -> `install_lpm_ci.sh`

What do you think?

Reply:
Alternatively, like in `install_cmake.sh` script I can add the following comment to the beginning of each one of the `install` scripts named above

> # This is suitable for CIs, not end users.

Reply:
yeah, maybe a comment is enough.

Reply:
done.